### PR TITLE
Priority: Update capture and purchase fields

### PR DESCRIPTION
**Pull request by priorityspreedly**
… fields

### CODE REVIEW

File: test/fixtures.yml [hunk 1]
```diff
@@ -854,9 +854,9 @@ plugnpay:
   password: PASSWORD
 
 priority:
-  key: SANDBOX_KEY
-  secret: SECRET
-  merchant_id: MERCHANT_ID
+  # key: SANDBOX_KEY
+  # secret: SECRET
+  # merchant_id: MERCHANT_ID
```
Comment (dsmcclain) [inline test/fixtures.yml:859]: Nit: these placeholder values don't need to be commented-out.

Their presence in this file is meant to communicate what credentials are needed to access the gateway, and my worry with commenting them out is that it communicates something different, like maybe that they are obsolete or unnecessary for this particular gateway.

File: lib/active_merchant/billing/gateways/priority.rb [hunk 2]
```diff
@@ -182,6 +183,17 @@ def add_type_merchant_purchase(params, merchant, is_settle_funds, options)
...
+        params['cardholderPresence'] = ''
+        params['deviceAttendance'] = ''
+        params['deviceInputCapability'] = ''
+        params['deviceLocation'] = ''
+        params['panCaptureMethod'] = ''
+        params['partialApprovalSupport'] = ''
+        params['pinCaptureCapability'] = ''
+        params['shipAmount'] = options[:shipAmount]
+        params['shipToCountry'] = options[:shipToCountry]
+        params['shipToZip'] = options[:shipToZip]
```
Comment (dsmcclain) [inline lib/active_merchant/billing/gateways/priority.rb:196]: Please change the keys in the `options` hash to snake_case, e.g. `params['shipToZip'] = options[:ship_to_zip]`

File: test/fixtures.yml (hunk 1 above)
Comment (priorityspreedly) [reply test/fixtures.yml:859]: This is a public repository. We have supplied credential via email for security

File: lib/active_merchant/billing/gateways/priority.rb [hunk 3]
```diff
@@ -182,6 +183,17 @@ def add_type_merchant_purchase(params, merchant, is_settle_funds, options)
         params['sourceZip'] = options[:billing_address][:zip] if options[:billing_address]
         params['taxExempt'] = false
         params['tenderType'] = 'Card'
+
```
[the remark this answers is not in the document]
Comment (priorityspreedly) [reply lib/active_merchant/billing/gateways/priority.rb]: removed

File: test/fixtures.yml (hunk 1 above)
Comment (priorityspreedly) [reply test/fixtures.yml:866]: Will email you the credentials.

File: lib/active_merchant/billing/gateways/priority.rb [hunk 4]
```diff
@@ -182,6 +183,17 @@ def add_type_merchant_purchase(params, merchant, is_settle_funds, options)
...
+        params['cardholderPresence'] = ''
+        params['deviceAttendance'] = ''
+        params['deviceInputCapability'] = ''
+        params['deviceLocation'] = ''
+        params['panCaptureMethod'] = ''
+        params['partialApprovalSupport'] = ''
+        params['pinCaptureCapability'] = ''
```
[the remark this answers is not in the document]
Comment (priorityspreedly) [reply lib/active_merchant/billing/gateways/priority.rb]: Updated code to populate these values.

File: lib/active_merchant/billing/gateways/priority.rb (hunk 2 above)
Comment (priorityspreedly) [reply lib/active_merchant/billing/gateways/priority.rb]: Changed to snake-case.

File: test/fixtures.yml (hunk 1 above)
Comment (dsmcclain) [reply test/fixtures.yml:866]: @priorityspreedly I apologize for the confusion here. We don't need you to share your credentials with us.

What I'm saying here is that we can just leave the code in this file as is, without commenting these fields out. Like this:
```
priority:
  key: SANDBOX_KEY
  secret: SECRET
  merchant_id: MERCHANT_ID
```

Comment (priorityspreedly) [reply test/fixtures.yml]: Removed # from priority: credentials.

**Comment by dsmcclain** [comment]
Merged as https://github.com/activemerchant/active_merchant/commit/da036cb6a84d584b344107ccb572533604b9136e with additional changes to conform to ActiveMerchant guidelines.

**Pull request closed without merging**